firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py

```diff
@@ -2,65 +2,54 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from marionette_driver import By
+from marionette_driver import By, Wait
 
 from firefox_ui_harness.decorators import skip_if_e10s
 from firefox_ui_harness import FirefoxTestCase
 
+from firefox_puppeteer.ui.windows import BrowserWindow
+
 
 class TestAboutPrivateBrowsing(FirefoxTestCase):
 
     def setUp(self):
         FirefoxTestCase.setUp(self)
-        self.pb_url = self.marionette.absolute_url('private_browsing/about.html?')
 
-    def tearDown(self):
-        self.marionette.close()
+        self.support_url = 'about:blank?'
+        self.pb_url = self.support_url + 'private-browsing'
```

Comment:
Both variables are used twice. I would agree to not bind ´support_url´ to the class but make it just a local variable. But we should really keep it for maintenance. You don't want to have to check the file which places hard-coded strings need to be replaced.

The question mark is part of the original URL which is stored in that pref. We overwrite that one, so we have to mimic that part. Its the parameter, which decides to load which site.